catalog/jobs/update.yaml

```diff
@@ -0,0 +1,6 @@
+description: "update giantswarm catalog"
+executor: architect
+steps:
+  - setup_remote_docker:
+      docker_layer_caching: true
+  - run: echo "hello orb"
```

Comment:
TODO I guess?

Reply:
Yes we will change this very soon in next PR.